examples/lighting-app/nrfconnect/prj.conf

```diff
@@ -14,123 +14,9 @@
 #    limitations under the License.
 #
 
-# Enable support for C++ language
```

Comment:
Pehaps a comment that the config is based on sample-defaults.conf and here are just overrides would be useful?

Reply:
Done